sdk/identity/azure-identity/azure/identity/_internal/get_token_mixin.py

```diff
@@ -62,8 +64,10 @@ def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken:
             For more information about scopes, see
             https://learn.microsoft.com/azure/active-directory/develop/scopes-oidc.
         :keyword str tenant_id: optional tenant to include in the token request.
+        :keyword str claims: not supported by this credential.
```

Comment:
This get_token method seems to be shared by some credentials that do support claims and some that presumably don't. But the ones that don't are not public or have a get_token override. So I say we add the docstring here:

```suggestion
        :keyword str claims: additional claims required in the token, such as those returned in a resource provider's
            claims challenge following an authorization failure.
```